.circleci/setup-heroku.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
I think this script is in the `heroku-deploy` docker image, so you don't need to include it here.

Reply:
(I think this is an innovation that might not have been back ported to all repos)

Reply:
Done